### ISSUE TITLE
Last Week in Pony - November 6, 2022

### ISSUE THREAD

**Issue by SeanTAllen** [ponylang/ponylang-website#796]

**Comment by rhagenson** [comment]
Highlight issues:

- https://github.com/ponylang/ponyc/issues/4051
  - `encode/base64` currently over-allocates space during decoding, wasting memory in a stdlib method. Someone taking this issue will have to think carefully about edge cases and the order of multiplication/division given the size of the encoded data. There is an existing [Zulip thread](https://ponylang.zulipchat.com/#narrow/stream/192795-contribute-to-Pony/topic/base64.20.28too.20long.3F.29) on the topic.
- https://github.com/ponylang/pony-tutorial/issues/446
  - Our Tutorial used a `Person` type to introduce type aliases, however in the process we made an oversight suggesting that every person has a home. This original oversight has been fixed, but we would like to rewrite the section to use something like animal classification instead. Ryan A. Hagenson, having worked on the Tutorial and animal classification before, is willing to help anyone who wishes to jump in on this issue. (Bonus points if the rewrite include lemur classification as that is what Ryan worked with in a previous life...oddly enough that classification work is what he was doing when he discovered Pony!)
  
Sean, if I do not get these committed to the branch I wanted to note them my "usual way" along with additional "why this issue" like was stated in Zulip.
